routers/api/v1/repo/file.go

```diff
@@ -40,21 +44,21 @@ func GetRawFile(ctx *context.APIContext) {
 	//   200:
 	//     description: success
 	if ctx.Repo.Repository.IsEmpty {
-		ctx.Status(404)
+		ctx.Status(http.StatusNotFound)
```

Comment:
Wasn't there a new method to return not found error with json error description?

Reply:
Yes, a recent PR of mine added it. Will need to merge master into the branch to get it.

https://github.com/go-gitea/gitea/blob/cac9e6e7605184f5679b1ebfbe5b5805191d9a53/modules/context/api.go#L147-L171

Reply:
Yeah, I merged with master and got those working.

Reply:
So I didn't know about that until I merged with master this morning to get my SDK changes in, but right away saw the new way. Let me know if I resolved conflicts right. Please also look to see @jolheiser if I need to update anything to match what you were doing.